javascript/chronometer.js

```diff
@@ -1,15 +1,19 @@
 class Chronometer {
```

Comment:
Hey Abbie :),

Please name the pull request according to the following format:

[FTAMS102020] <name>